gslib/commands/du.py

```diff
@@ -214,15 +232,12 @@ def RunCommand(self):
           if a == '-':
             f = sys.stdin
           else:
-            f = open(a, 'r')
+            f = open(a, 'r', encoding=UTF8)
```

Comment:
I didn't think you could supply `encoding` as a kwarg for Python 2's `open` function? How does this work on Py2?

Reply:
That's true, I could have sworn I ran Py2 test with it coming up clean, but I guess I missed it. Changes in #674 to fix.